codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -3265,21 +3265,16 @@ loadAllProjects =
     |]
 
 -- | Load all projects whose name matches a prefix.
-loadAllProjectsBeginningWith :: Text -> Transaction [Project]
-loadAllProjectsBeginningWith prefix =
-  -- since we are not likely to many projects, we just get them all and filter in Haskell. This seems much simpler than
-  -- running a LIKE query, and dealing with escaping, case sensitivity, etc
```

Comment:
I'm guessing maybe it just wasn't super clear how to use `globEscape` when this was written, but it's really not tricky to use SQLite for this so I swapped it in.

GLOB is case-sensitive, which matches the existing `Text.isPrefixOf` behaviour.